internal/pkg/config/output.go

```diff
@@ -180,7 +181,7 @@ func (c *Output) Validate() error {
 
 func makeURL(defaultScheme string, defaultPath string, rawURL string, defaultPort int) (string, error) {
 	if defaultScheme == "" {
-		defaultScheme = "http"
+		defaultScheme = defaultProtocol
```

Comment:
nit: would prefer maybe something like "schemeHTTP" constant instead "defaultProtocol", easier to to understand without scrolling up to see the defaultProtocol constant value